현정/6주차_11650.java

```diff
@@ -0,0 +1,49 @@
+import java.io.*;
+import java.util.*;
+
+public class BJ_11650 {
+    static int n;
+    static Point[] arr;
+
+    static class Point{
+        int y; int x;
+        public Point(int y, int x){
+            this.y = y;
+            this.x = x;
+        }
+
+    }
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st;
+
+        n = Integer.parseInt(br.readLine());
+        arr = new Point[n];
+        for (int i = 0; i < n; i++) {
+            st = new StringTokenizer(br.readLine());
+            int y = Integer.parseInt(st.nextToken());
+            int x = Integer.parseInt(st.nextToken());
```

Comment:
x, y 값이 반대가 된 것 같네요. (x, y)가 되도록 나머지 부분들도 수정하면 좋을 것 같아요!!
```suggestion
            int x = Integer.parseInt(st.nextToken());
            int y = Integer.parseInt(st.nextToken());
```